lang/cs/Org.Apache.REEF.Bridge.CLR/Org.Apache.REEF.Bridge.CLR.DotNet.csproj

```diff
@@ -36,7 +36,7 @@ under the License.
     <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonVersion)" />
```

Comment:
Need to add build.DotNet.targets to this file,

Reply:
Please make the above changes so I can test against them.